<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>241</width>
        <height>491</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxID">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Voltaje</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxV">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Posición X</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinBoxX">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Posición Y</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="spinBoxY">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSpinBox" name="spinBoxR">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QSpinBox" name="spinBoxG">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="spinBoxB">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonAI">
         <property name="text">
          <string>Agregar Inicio</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonAF">
         <property name="text">
          <string>Agregar Final</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonGuardar">
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonRecuperar">
         <property name="text">
          <string>Recuperar</string>
         </property>
        </widget>
       </item>
       <item row="26" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonM">
         <property name="text">
          <string>Mostrar Neuronas</string>
         </property>
        </widget>
       </item>
       <item row="27" column="0" colspan="2">
        <widget class="QLabel" name="labelMostrar">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="28" column="0" colspan="2">
        <widget class="QLabel" name="labelMostrar2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="25" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Mostrar Gráficamente</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="23" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="22" column="0">
        <layout class="QFormLayout" name="formLayout"/>
       </item>
       <item row="20" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonO">
         <property name="text">
          <string>Ordenar Lista por ID</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonO2">
         <property name="text">
          <string>Ordenar Lista por Voltaje</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>711</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>510</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Neuronas Guardadas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRegresar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetMostrar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>711</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Voltaje</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posición X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posición Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Red</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Green</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Blue</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButtonBN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar Neurona por ID</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetMostrar2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Voltaje</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posición X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posición Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Red</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Green</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Blue</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>430</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEditMostrar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>711</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxBN">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
     <zorder>plainTextEditMostrar</zorder>
     <zorder>label</zorder>
     <zorder>pushButtonRegresar</zorder>
     <zorder>pushButtonBN</zorder>
     <zorder>tableWidgetMostrar2</zorder>
     <zorder>label_2</zorder>
     <zorder>spinBoxBN</zorder>
     <zorder>tableWidgetMostrar</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
